server/platform/shared/web/files.go

```diff
@@ -48,6 +49,23 @@ func WriteFileResponse(filename string, contentType string, contentSize int64, l
 		}
 	}
 
+	http.ServeContent(w, r, filename, lastModification, fileReader)
+}
+
+func WriteStreamResponse(w http.ResponseWriter, r io.ReadCloser, filename string, contentType string, forceDownload bool) error {
```

Comment:
At first glance, I don't know which one of these now two methods I should be using: would it make sense to converge on one, and change the callers to conform to the stream syntax? (0/5)

Reply:
The issue is that `http.ServeContent` does a lot of extra work around range and mulitparts, so I definitely don't want to mess with that.
But the broader point is good -- I've included a comment to help differentiate the two, but I'm open to suggestions to make them better.